tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md

```diff
@@ -115,29 +115,38 @@ The azure-sdk team regularly update the image associated with the `latest` tag.
 To ensure that your local copy is up to date, run:
 
 ```powershell
-> docker pull azsdkengsys.azurecr.io/engsys/testproxy-lin:latest
+> docker pull azsdkengsys.azurecr.io/engsys/test-proxy:latest
 ```
 
 ## Command line arguments
 
 This is the help information for test-proxy. It uses the nuget package [`CommandLineParser`](https://www.nuget.org/packages/CommandLineParser) to parse arguments.
 
+The test-proxy executable fulfills one of two primary purposes:
+
+1. The test-proxy server (the only option up to this point)
+2. `asset-sync` push/restore/reset.
```

Comment:
```suggestion
2. [`asset-sync`](#asset-sync-retrieve-external-test-recordings) push/restore/reset.
```

Reply:
If you take this, the backticks might need to surround the whole thing? I forget how markdown resolves links + code blocks together.

Reply:
Tested locally and it works just fine. Takin it.